compiler/qsc_linter/src/tests.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{linter::ast::run_ast_lints, Lint, LintLevel};
+use expect_test::{expect, Expect};
+use qsc_ast::{
+    assigner::Assigner,
+    ast::{self, NodeId},
+    mut_visit::MutVisitor,
+};
+use qsc_data_structures::language_features::LanguageFeatures;
+
+#[test]
+fn multiple_lints() {
+    check_ast(
+        "let x = ((1 + 2)) / 0;",
+        &expect![[r#"
+        [
+            SrcLint {
+                source: "((1 + 2)) / 0",
```

Comment:
That can be done with another Lint that matches against unnecessary parentheses instead of extra set of parentheses. For example, Clippy has two lints, one called `DoubleParens` and another called `NeedlessParens`. We could follow the same pattern here.

Reply:
Added `NeedlessParens` lint.